### DISCUSSION TITLE
How to get started

### DISCUSSION THREAD

**Discussion by ScottinOkla** [cityssm/lot-occupancy-system#83] [General]
Our organization manages a small cemetery in Oklahoma.  Our records are all still kept in paper books that date back to the early 1900s.  I am looking to modernize our records, and make them easily searchable and accessible from anywhere.  I know that the first step will be to digitize our records, but after that, I would love to make a map that is searchable and can point visitors towards their loved ones.  This would also be handy for visualizing empty plots.
I consider myself technical, but have zero experience programming.
What would be the best path forward in our journey?

**Comment by dangowans** [comment]
Hi @ScottinOkla .  Thanks for reaching out.  Our situations have some similarities for sure.

Our records are for the most part digitized, however they are in a legacy, end-of-life system.  Our goal is to move them to something current and easier to maintain.

We're trying to tackle our project in phases.

**Phase 1** _(Current phase)_
Move over all of the data from the legacy cemetery system into this new system.

**Phase 2**
Build interactive maps of our cemeteries.  We too want to see occupied blocks of our cemeteries versus those that are available.

**Phase 3**
Offer a public facing system to the general public.

My recommendation for you is to start your digitizing process in a spreadsheet.  By splitting up the information into clean, well defined columns, you will more easily be able to move that data into other systems like this one.

A sample sheet might look like this.

<table>
<thead>
<tr>
<th>Cemetery Name</th>
<th>Plot Number</th>
<th>Interment Number</th>
<th>Owner</th>
<th>Purchase Date</th>
<th>Deceased Name</th>
<th>Death Date</th>
<th>Interment Date</th>
<th>Comments</th>
</tr>
</thead>
<tbody>
<tr>
<td>Greenwood</td>
<td>GW-A-100-5</td>
<td>2</td>
<td>Joe Smith</td>
<td>2023-07-01</td>
<td>Jane Smith</td>
<td>2023-07-01</td>
<td>2023-07-04</td>
<td>Interred with husband, Bill Smith.</td>
</tr>
</tbody>
</table>

You may have more or less columns to track.  For example, you may track the type of vault, whether the body is cremated or not, owner contact details, etc.

Two very important things to keep in mind.

- Be as consistent as possible.  For example, a column called "Is Cremated" should have either "TRUE" or "FALSE" in it, not "YES", "True", "1", "TRUE", "TURE", etc.

- Avoid grouping together key details in a comments column that you may want to filter by or report on.  For example, if you are regularly putting the name of the funeral home you are working with in the comments column, you may be better to have a separate "Funeral Home" column.

I hope this helps.  Good luck with your digitization project.
